src/routers/xcm/Router.sol

```diff
@@ -25,13 +26,70 @@ contract ConnectorXCMRouter {
     using TypedMemView for bytes29;
     using ConnectorMessages for bytes29;
 
+    /// --- Properties ---
     ConnectorLike public immutable connector;
+    address immutable centrifugeChainOrigin;
+    uint8 immutable centrifugeChainConnectorsPalletIndex;
+    uint8 immutable centrifugeChainConnectorsPalletHandleIndex;
+    XcmWeightInfo xcmWeightInfo;
 
-    address centrifugeChainOrigin;
+    /// --- Storage ---
+    /// Auth storage
```

Comment:
I think this is a bit redundant
```suggestion
```

Reply:
I will totally remember what `wards` is 😛